rosdep/python.yaml

```diff
@@ -1843,6 +1843,19 @@ python-graphviz-pip:
 python-gridfs:
   debian: [python-gridfs]
   fedora: [python-pymongo-gridfs]
+python-grip:
+  debian:
+    pip:
+      packages: [grip]
+  fedora:
+    pip:
+      packages: [grip]
+  gentoo:
+    pip:
+      packages: [grip]
+  ubuntu:
+    pip:
+      packages: [grip]
```

Comment:
It looks like this is available as a system package on Ubuntu: https://packages.ubuntu.com/search?keywords=grip&searchon=names&suite=all&section=all .  Please change it to use that package instead.

Reply:
Thank you! I updated.